python-dictionaries/dictionary-exercise.py

```diff
@@ -0,0 +1,15 @@
+def making_dict(n):
+    binary_dict = {}
+    for num in range(n):
+        bin_dict_str = ''
+        temp_remain = num
```

Comment:
You should use the temp variable that you set from the previously defined variable as the one that gets modified later